artemis-commons/src/main/java/org/apache/activemq/artemis/utils/ByteUtil.java

```diff
@@ -389,6 +389,16 @@ private static boolean equalsUnsafe(final byte[] left,
       return true;
    }
 
+   /**
+    * This ensure a more exact resizing then {@link ByteBuf#ensureWritable(int), if needed.<br>
```

Comment:
this seems problematic:
/home/runner/work/activemq-artemis/activemq-artemis/artemis-commons/src/main/java/org/apache/activemq/artemis/utils/ByteUtil.java:393: error: unexpected text after parenthesis
Error:      * This ensure a more exact resizing then {@link ByteBuf#ensureWritable(int), if needed.<br>
Error:                                               ^